ansible/roles/debops.phpipam/defaults/main.yml

```diff
@@ -10,240 +10,328 @@
 # .. include:: ../../../includes/global.rst
 
 
-# .. envvar:: phpipam_mode [[[
+# Basic configuration options [[[
+# -------------------------------
+
+# .. envvar:: phpipam__mode [[[
 #
 # What parts of the "phpIPAM stack" to enable:
 #
 # - ``webui``: configure the Web interface
 #
 # - ``scripts``: configure phpipam-scripts
 #
-phpipam_mode: [ 'webui', 'scripts' ]
+phpipam__mode: [ 'webui', 'scripts' ]
 
                                                                    # ]]]
-# ---- Basic configuration options ----
-
-# .. envvar:: phpipam_dependencies [[[
+# .. envvar:: phpipam__dependencies [[[
 #
 # Should phpipam role manage it's own dependencies?
-phpipam_dependencies: True
+phpipam__dependencies: True
```

Comment:
This variable was relevant when the role dependencies were in `meta/main.yml` and therefore "hard" dependencies. Now that they are in a playbook, and playbooks can be written by the users, this variable can be removed.

Reply:
Done